resources/builder.inc.sh

```diff
@@ -485,21 +514,22 @@ _builder_execute_child_script() {
     dep_module="$builder_dep_parent"
   fi
 
+  # Note: we do not pass '$target' to the child script, because it is used here
+  # only for reporting; the target was used to select the correct script in
+  # _builder_execute_child()
+
```

Comment:
I'm not sure this comment still makes sense since we no longer set `$target` variable.